terraform/dev/terragrunt.hcl

```diff
@@ -10,6 +10,8 @@ generate "dev_tfvars" {
   path              = "dev.auto.tfvars"
   if_exists         = "overwrite"
   disable_signature = true
+  cloudfront        = true
```

Comment:
These need to be after the first EOF, beside `alb_name` and `service_names`.

Reply:
made changes